src/Util/DirectoryUtil.php

```diff
@@ -42,13 +42,35 @@ public static function stripAncestor(string $path, string $ancestor): string
      * Strips the trailing slash (directory separator) from a given path.
      *
      * @param string $path
-     *   Any path, absolute or relative, existing or not.
+     *   Any path, absolute or relative, existing or not. Empty paths and device
+     *   roots will be returned unchanged--but note that Windows UNC root path
```

Comment:
What is UNC?

Reply:
Universal Naming Convention: https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats#unc-paths. Example: `\\Server\Share\Directory\File.txt`. I had never heard of it until I started reading up on Windows file paths for this method, and to be honest, I wasn't thrilled about it. 😅